src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx

```diff
@@ -130,12 +130,9 @@
     <value>A project for creating a command-line application that can run on .NET Core on Windows, Linux and MacOS.</value>
   </data>
   <data name="5" xml:space="preserve">
-    <value>XProj Files (*.xproj);*.xproj</value>
```

Comment:
Yes the C# and VB [templates](https://github.com/dotnet/sdk/blob/master/src/Templates/ProjectTemplates/VisualBasic/.NETCore/VisualBasicClassLibrary/ClassLibrary.vstemplate) need to be adjusted.